Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/azure/serverless/spark/models/CreateSparkBatchJobParameters.java

```diff
@@ -0,0 +1,440 @@
+/**
```

Comment:
These classes "seem" autogenerated,
1) if they are autogenerated from autorest, we should add either a header telling that its generated or the folder in which it is placed should be named generated.

2) if they are not autogenerated, can you  reply as to why they are not? does autorest cause any problem?

Reply:
The API is auto-generated first and then modified manually. We did some modification since:
1. `autorest` won't add `@JsonIgnoreProperties(ignoreUnknown = true)` for us but we need this feature
2. Some serializaion code need to be added to leverage existing code. e.g. Some class in the API need to implement interfact `IConvertible` to help serialize an object to XML-format or Json-format.